portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +79,16 @@ const handleQuantityButton = function() {
         getComments(max);
     });
 };
+
+const deleteAllComments = function() {
+    const request = new Request('/delete-data', {method: 'POST'});
+    fetch(request).then(response => {
+        getComments();
```

Comment:
If all the comments are deleted, do you need to fetch them? Maybe you can just fire the fetch and not do anything after e.g.

```javascript
fetch(request);
```